etc/config/go.amazon.properties

```
@@ -75,7 +75,10 @@ compiler.gl1150.semver=1.15
 compiler.gl1150.goroot=/opt/compiler-explorer/golang-1.15/go
 compiler.gl1160.exe=/opt/compiler-explorer/golang-1.16/go/bin/go
 compiler.gl1160.semver=1.16
-compiler.gl1160.goroot=/opt/compiler-explorer/golang-1.16/go
+compiler.gl1170.goroot=/opt/compiler-explorer/golang-1.16/go
+compiler.gl1170.exe=/opt/compiler-explorer/golang-1.17/go/bin/go
+compiler.gl1170.semver=1.17
+compiler.gl1170.goroot=/opt/compiler-explorer/golang-1.16/go
```

This should be probably be:

```suggestion
compiler.gl1170.goroot=/opt/compiler-explorer/golang-1.17/go
```